### PR TITLE
neovim: update to 0.6.0

### PR DESCRIPTION
**Pull request by gpanders**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.0.1 21A559 arm64
Xcode 13.1 13A1030d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: editors/neovim/Portfile [hunk 1]
```diff
@@ -22,9 +22,9 @@ long_description \
 
 homepage                https://neovim.io
 
-checksums               rmd160  de32b78f5172360c46e5b42864b1bce44fb17f9c \
-                        sha256  e2babf3a8aac890f9299578f017857537e8d51035d87d8934a5fcd2ab5d5a131 \
-                        size    10348033
+checksums           rmd160  ef32e586845e6ec9636094a06fc59ef90aef62fb \
```
Comment (l2dy) [inline editors/neovim/Portfile:25]: Please keep lines aligned within the Portfile.

**Comment by esn89** [comment]
@gpanders 

Curious how you got the rmd160 sum.  Did you run:
`openssl dgst -rmd160` on https://github.com/neovim/neovim/releases/download/v0.6.0/nvim-macos.tar.gz ?

**Comment by gpanders** [comment]
I used `port -v checksum`.

**Review by l2dy — APPROVED** [review]

**Comment by gpanders** [comment]
Anything else I need to do to get this merged?

**Comment by l2dy** [comment]
Merged. Thank you!

**Pull request merged**